src/backend/storage/smgr/smgr.c

```diff
@@ -137,7 +137,7 @@ smgr(BackendId backend, RelFileNode rnode)
  *		This does not attempt to actually open the underlying file.
  */
 SMgrRelation
-smgropen(RelFileNode rnode, BackendId backend)
+smgropen(RelFileNode rnode, BackendId backend, char relpersistence)
```

Comment:
It is not mentioned somewhere what relpersistence == 0 means?
Does it main persistence of relation is not known at this moment?

Reply:
Right. There is an explanation at the top of `pagestore_smgr.c`, but I'll add a comment here, too.